lib/statsd.js

```diff
@@ -227,11 +242,53 @@ Client.prototype.send = function (stat, value, type, sampleRate, tags, callback)
 };
 
 /**
+ *
+ * @param message {String}
+ */
+Client.prototype.enqueue = function(message){
+  this.buffer += message + "\n";
```

Comment:
That would defeat the purpose.  We aggregate packets here to reduce the number of calls to socket.send, which is an expensive call.